### PR TITLE
Caching and read fixes

### PR DESCRIPTION
**Pull request by codinpsycho**
* Update only valid keys in DB, correct ones were getting overwritten in DB
* Cache Fix
* Putting reads and cache requests in fastq to throttle concurrency.

### CODE REVIEW

File: src/dal/nft-repo/nft-info.accessor.ts [hunk 1]
```diff
@@ -8,31 +8,50 @@ export class NftInfoAccessor {
...
 
   public async insert(data: NftInfo): Promise<any> {
-    const result = await this.NftInfoDataModel.create(data);
-    return result;
+    try {
+      const result = await this.NftInfoDataModel.create(data);
+      return result;
+    } catch (err) {
+      console.log(err);
+    }
```
Comment (vgvishesh) [inline src/dal/nft-repo/nft-info.accessor.ts:16]: Why are we not throwing errors? Application later will not know if this failed, the logic that depends of unsuccessful db updates can fail.

File: src/modules/helper/remote-data-fetcher/data-fetcher.service.ts [hunk 2]
```diff
@@ -271,28 +330,34 @@ export class RemoteDataFetcherService {
...
-			await this.nftInfoAccessor.insert(nftResponse as NftInfo);
-			Object.assign(nftResponse, { cached_image_uri: nftResponse.image_uri });
-		}
-		if (nftResponse.image_uri) {
-			const nftCacheEvent = new NftCacheEvent(nftResponse.image_uri);
-			this.eventEmitter.emit('nft.cache', nftCacheEvent);
-		}
-		return nftResponse;
-	}
+  async getNftDetails(
```
Comment (vgvishesh) [inline src/modules/helper/remote-data-fetcher/data-fetcher.service.ts:342]: This function should be in `nft-db-sync.service.ts`

Comment (codinpsycho) [reply src/modules/helper/remote-data-fetcher/data-fetcher.service.ts:342]: Hmm, it was already here. It is being used in listing service which has access to data-fetcher service. There is no direct access to Db Sync service as it is only listens to emitted events. I think this function itself doesn't feel right.
But that's a diff thing, not related to this change.

File: src/dal/nft-repo/nft-info.accessor.ts (hunk 1 above)
Comment (codinpsycho) [reply src/dal/nft-repo/nft-info.accessor.ts:16]: will add in the next PR

**Pull request merged**